hack/observability/promtail/values.yaml

```diff
@@ -2,4 +2,16 @@
 
 config:
   # publish data to loki
-  lokiAddress: http://loki:3100/loki/api/v1/push
\ No newline at end of file
+  lokiAddress: http://loki:3100/loki/api/v1/push
+
+  snippets:
+    pipelineStages:
+    # Parse cluster and machine to make them available as labels.
+    - cri: { }
+    - json:
+        expressions:
+          cluster: cluster
+          machine: machine
+    - labels:
+        cluster:
+        machine:
```

Comment:
This adds the cluster/machine label if they are part of the log line

TODO: check what happens with text logging format

Reply:
Verified with text. The labels are obviously not parsed, but the log lines still make it to Loki!